src/common/RootProviders.tsx

```diff
@@ -1,3 +1,4 @@
+import { RootSiblingParent } from 'react-native-root-siblings';
```

Comment:
I see this lib in yarn.lock, but not package.json, which seems wrong.

Also, can you explain what this lib is doing in relation to the `DetectNetworkConnectionProvider`?

Reply:
> I see this lib in yarn.lock, but not package.json, which seems wrong.

`react-native-root-siblings` is a dependency of `react-native-root-toast` so it is installed when `react-native-root-toast` is installed.

> Also, can you explain what this lib is doing in relation to the DetectNetworkConnectionProvider?

RootSiblingParent is required to utilize 'root' components, i.e., the toast message which can be invoked via API anywhere. It is one of the first steps of setup for [react-native-root-toast](https://github.com/magicismight/react-native-root-toast).

As for why the `react-native-root-toast` library:
1. Mentioned in expo docs [here](https://docs.expo.dev/ui-programming/react-native-toast/).
2. The only toast library out of three that I tried that worked as expected without errors

Reply:
If we stick to this lib, we can't/shouldn't rely on a dep of dep like that and will need to add `react-native-root-siblings` as a package.json entry / install.  As noted in another comment, I'm hopeful that other lib (react-native-toast-message) prototypes well, replaces this one in this PR, and then we won't need this.